VespaGASTests/ViewModels/GasCalculatorViewModelTests.swift

```diff
@@ -1,9 +1,158 @@
 import XCTest
+import RxSwift
 @testable import VespaGAS
 
 class GasCalculatorViewModelTests: XCTestCase {
-    func testViewModelIsNotNil() {
-        let viewModel = GasCalculatorViewModel()
-        XCTAssertNotNil(viewModel)
+    let disposeBag = DisposeBag()
+
+    //MARK: oilMixObservable
+    func testOilMixValueObservableIsInitialisedWith0() {
+        let expectation = self.expectationWithDescription("Oil Observable should start with 0ml")
+
+        let viewModel = GasCalculatorViewModel(
+            gasObservable: Observable.empty(),
+            oilPercentageObservable: Observable.empty()
+        )
+
+        _ = viewModel.oilMixObservable.subscribeNext { (oilML) in
+            XCTAssertEqual("0ml", oilML)
+            expectation.fulfill()
+            }.addDisposableTo(disposeBag)
+
+        self.waitForExpectationsWithTimeout(0.1, handler: nil)
```

Comment:
RxBlocking is highly recommended for tests, I'd argue it should be required for all new projects (especially as the blocking stuff is in RxJava main lib)